lib/connectors_async/job_runner.rb

```diff
@@ -18,10 +18,11 @@ def initialize(max_threads:)
 
     def start_job(job:, connector_class:, cursors:, modified_since:, access_token:)
       @pool.post do
+        Time.zone = ActiveSupport::TimeZone.new('UTC') # bah Time.zone should be init for each thread
```

Comment:
I would probably create a `init_thread` function where we can do all the initialization required (even if it's just one for now, for extra clarity